examples/grid_packing/main.rs

```diff
@@ -0,0 +1,54 @@
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.grid-packing"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(application: &gtk::Application) {
+    // Create a new window, set its title and default size
+    let window = gtk::ApplicationWindow::new(application);
+    window.set_title(Some("Grid Packing"));
+    window.set_default_size(200, 120);
+
+    // Here we construct the grid that is going contain our buttons.
+    let grid = gtk::GridBuilder::new()
+        .margin_start(6)
+        .margin_end(6)
+        .margin_top(6)
+        .margin_bottom(6)
+        .halign(gtk::Align::Center)
+        .valign(gtk::Align::Center)
+        .row_spacing(6)
+        .column_spacing(6)
+        .build();
+
+    // Add the grid in the window
+    window.set_child(Some(&grid));
+
+    // Create the first button, make it expand and put it into the grid at (0, 0)
```

Comment:
```suggestion
    // Create the first button and put it into the grid at (0, 0)
```
You don't really make any of the buttons expand? the same thing about the two other buttons